src/collection.js

```diff
@@ -342,6 +342,22 @@ module.exports = function(options) {
     })
   }
 
+  const allVersions = callback => {
+    preCommand(() => {
+      repo.getTags('v_*_', tags => {
+        if (!tags) return callback(undefined)
+
+        const versions = tags
+          .map(t => t.match(/v_(.*)_/))
+          .filter(t => t)
+          .map(t => t[1].split(/\./).map(Number))
+          .map(t => ({ major: t[0], minor: t[1] }))
```

Comment:
`([major, minor]) => ({major, minor})` ?

Reply:
i like it